python/lsst/ip/isr/calibType.py

```diff
@@ -91,15 +93,42 @@ def __str__(self):
     def __eq__(self, other):
         """Calibration equivalence.
 
-        Subclasses will need to check specific sub-properties.  The
-        default is only to check common entries.
+        Running ``calib.log.setLevel(0)`` enables debug statements to
+        identify problematic fields.
         """
         if not isinstance(other, self.__class__):
+            self.log.debug("Incorrect class type: %s %s", self.__class__, other.__class__)
             return False
 
         for attr in self._requiredAttributes:
-            if getattr(self, attr) != getattr(other, attr):
+            attrSelf = getattr(self, attr)
```

Comment:
Yes.  Since they were doing essentially the same kind of checks, it made sense to collect them in the base class `__eq__` so any future calibs will have a functional equivalence with no extra work.